README.md

```diff
@@ -58,6 +58,12 @@ The RoboCup @Home Organizing Committee is always looking for volunteers to assis
 
 Contributions and improvements to the rulebook are always welcome in the form of pull requests.
 
+### License
+
+The content of this project itself is licensed under the Creative Commons Attribution Share Alike 4.0 International license, and the underlying source code used to format and display that content is licensed under the GPL v2 license.
+
+Note that neither of these licenses grant rights to the RoboCup trademark. If you would like to host a sanctioned event, please contact the league organizers.
```

Comment:
> Note that neither of these licenses grant rights to the RoboCup trademark.

... or to the use of the RoboCup logo.

> If you would like to host a sanctioned event, please contact the league organizers.

If you would like to host a sanctioned event including, but not limited to competitions, workshops, conferences, and educational projects, you must contact the RoboCup Federation.

I'm dubtious about this previous line. While RoboCup and the RoboCup logo are registered trademarks, I don't think the same is valid for RoboCup@Home and our logo. Furthermore, *the league organizers* refers to the @Home league organization (e.g. any OC member), while not to the president of the RoboCup Federation, so we need to be careful here.

Perhaps @iocchi can tell us better.